test/MSISonar_TEST.cc

```diff
@@ -14,6 +14,15 @@
 // OpenCV includes
 #include <opencv2/opencv.hpp>
 
+#if GAZEBO_MAJOR_VERSION >=8
+namespace gazebo {
+
+namespace math {
+  using Vector3 = ignition::math::Vector3d;
+};  // math
+};  // gazebo
+#endif
```

Comment:
I didn't like this strategy. It is confusing to read `gazebo::math::Vector3` in the code.